lib/active_model_serializers/adapter/attributes.rb

```diff
@@ -5,7 +5,10 @@ module Adapter
     class Attributes < Base
       def initialize(*)
         super
-        instance_options[:fieldset] ||= ActiveModel::Serializer::Fieldset.new(fields_to_fieldset(instance_options.delete(:fields)))
```

Comment:
`instance_options[:fieldset]` is always  `nil` as it's not listed in `ADAPTER_OPTION_KEYS`

Reply:
excluded